package.json

```diff
@@ -22,6 +22,7 @@
     "docs:build": "rimraf dist && yarn docz build",
     "docs:deploy": "npm run docs:build && gh-pages -d dist",
     "prepublishOnly": "npm-run-all build",
+    "preview": "yalc publish --push",
```

Comment:
Not sure if this command already does this and certainly doesn't need to be in this PR but having `preview` watch source files and run `publish` automagically would be sweet

Reply:
A watch command has been added using `nodemon`.